test/e2e/cli_test.go

```diff
@@ -108,6 +112,26 @@ func (s *CLISuite) TestRoot() {
 			}
 		})
 	})
+
+	var createdWorkflowName string
+	s.Given(s.T()).CronWorkflow("@testdata/basic.yaml").
+		When().
+		CreateCronWorkflow().
+		RunCli([]string{"submit", "--from", "CronWorkflow/test-cron-wf-basic"}, func(t *testing.T, output string, err error) {
+			assert.NoError(t, err)
```

Comment:
would it be better to create an `ExpectCronWorkflow(..)` func and capture it in that? more robust and I've created a few examples in the archiving code